<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>700</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#FCF6F5;</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>325</x>
     <y>80</y>
     <width>400</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>26</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#990011</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;History&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="menu_widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1050</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#menu_widget{
border-top: 1px solid rgb(0, 0, 0);
border-bottom: 1px solid rgb(0, 0, 0);

background-color: rgb(227, 227, 229)
} 


#menu_widget QPushButton
{

background-color: transparent;
}



#menu_widget QLabel{
 background-color: transparent;
}




#menu_widget QPushButton:checked
{

padding-top: 3px;
border: 3px solid black;
border-left: none;
border-right: none;
border-top: none;
}





</string>
   </property>
   <widget class="QLabel" name="home_label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>37</y>
      <width>60</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Home</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="homeButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>-12</y>
      <width>60</width>
      <height>73</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../Downloads/home.png</normaloff>../../Downloads/home.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>26</width>
      <height>26</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="history_label">
    <property name="geometry">
     <rect>
      <x>495</x>
      <y>37</y>
      <width>60</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>History</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="historyButton">
    <property name="geometry">
     <rect>
      <x>495</x>
      <y>-8</y>
      <width>60</width>
      <height>69</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../Downloads/histroy.png</normaloff>../../Downloads/histroy.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>26</width>
      <height>26</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="settings_label">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>37</y>
      <width>60</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Help&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="settingsButton">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>-12</y>
      <width>60</width>
      <height>73</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../Downloads/settings.png</normaloff>../../Downloads/settings.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>26</width>
      <height>26</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>180</y>
     <width>980</width>
     <height>481</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QListWidget {
    font-family: &quot;Century Gothic&quot;;
    font-size: 22px;
    font-weight: bold;
}

QListWidget::item {
    font-family: &quot;Century Gothic&quot;;
    font-size: 22px;
    font-weight: bold;
}

QListWidget::item {
    border-bottom: 1px solid black;
    padding: 10px; 
}


QListWidget::item:hover {
    background-color: lightblue; 
}

QListWidget::item:selected {
    background-color: lightblue; 
	color:#000000;
}</string>
   </property>
   <item>
    <property name="text">
     <string>New Item</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
